packages/api/src/external/commonwell/patient.ts

```diff
@@ -136,6 +136,33 @@ export async function create(
   }
 }
 
+export async function get(
+  patient: Patient,
+  facilityId: string
+): Promise<CommonwellPatient | undefined> {
+  const { debug } = Util.out(`CW get - M patientId ${patient.id}`);
+
+  const cwEnabled = await validateCWEnabled({
+    patient,
+    facilityId,
+    debug,
+  });
+  if (!cwEnabled) return undefined;
```

Comment:
I know this function isnt being used yet but it would help to add a log here so we can better diagnose if we didnt get anyting when its in prod

Reply:
The other endpoints don't log directly, they all do through `validateCWEnabled()`.